Wordle.Api/Wordle.Api/Identity/IdentitySeed.cs

```diff
@@ -43,5 +48,77 @@ private static async Task SeedAdminUserAsync(UserManager<AppUser> userManager)
                 await userManager.AddToRoleAsync(user, Roles.Admin);
             }
         }
+
+        // Seed Admin User
+        if (await userManager.FindByEmailAsync("nmarsee@ewu.edu") == null)
+        {
+            AppUser user = new AppUser
```

Comment:
No master of universe claim implemented, but when it is update this so the seeded users can be authorized to add/delete words :)

Reply:
I think we did implement MasterOfTheUniverse, right? Unless I'm missing something, see line 67:
       await userManager.AddClaimAsync(user, new Claim(Claims.MasterOfTheUniverse, "true"));
As well as the policies.cs and claims.cs.
Adding or removing words requires this claim, as shown in WordController.cs

Reply:
we did, i dont think is on any of your testing accounts